qiita_db/metadata_template/prep_template.py

```diff
@@ -432,6 +404,16 @@ def create_qiime_mapping_file(self, prep_template_fp):
         ------
         ValueError
             If the prep template is not a subset of the sample template
+        QiitaDBWarning
+            If the QIIME-required columns are not present in the template
+
+        Notes
+        -----
+        We cannot ensure that the QIIME-required columns are present in the
+        metadata map. However, we have to generate a QIIME-compliant mapping
+        file. Since the user may need a QIIME mapping file, but not these
+        QIIME-required columns, we are going to create them and
+        populate them with the value XXQIITAXX.
         """
         rename_cols = {
```

Comment:
Thanks, while changing this I noticed that the prep template already created in the database is not fully correct. I'm changing it and it will probably require a couple of more changes in the rest of the tests.